docs/user_guide/HOWTO/IPA_Clients.rst

```diff
@@ -0,0 +1,105 @@
+.. _howto-ipa_clients:
+
+HOWTO Enroll Hosts into an IPA Domain
+=====================================
+
+Host should be able to join an IPA domain with a few catches. SIMP already uses
+the login stack that IPA uses (PAM, SSSD), but it also optionally manages the
+same resources that IPA provides automation for, like:
+
+=========== =============================== ===============
+Technology  Related SIMP features           Related tickets
+=========== =============================== ===============
+``sudoers`` simp-simp and simp-sudo modules `SIMP-4898`_
+``autofs``  optional simp-simp_nfs module   `SIMP-4168`_
+``krb5``    optional simp-krb5 module       `SIMP-4167`_
+=========== =============================== ===============
+
+These features may work in the future, but logins via SSSD or LDAP should work
+without issue, now.
+
+
+Adding clients
+--------------
+
+Adding clients requires two steps:
+
+#. Add the hosts on the IPA server, setting a one time password
+#. Running ``ipa-client-install`` on the host, using the password generated in
```

Comment:
Yep, it needs to be actually added into the how-tos -- where do you think we should put it? Is there any order to it now?
When i randomly put the link to this at the end, after the IMA instructions, i noticed that the last IMA numbered instruction had reset to 1, so that may need to be fixed, too...  http://simp.readthedocs.io/en/master/user_guide/HOWTO/Manage_TPM.html